lang/java/avro/src/test/java/org/apache/avro/TestFixed.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.avro;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestFixed {
+
+
+  @Test
+  public void testFixedDefaultValueDrop() {
```

Comment:
The main reason I created a separate unit test is that the issue is not only in the schema builder, it is also in the JacksonUtils.java...